cordova-lib/src/platforms/platforms.js

```diff
@@ -102,5 +102,10 @@ function getPlatformProject(platform, platformRootDir) {
 }
 
 module.exports = platforms;
-module.exports.getPlatformProject = getPlatformProject;
-module.exports.PlatformProjectAdapter = PlatformProjectAdapter;
+
```

Comment:
Ok, got it, looks about right, but for some reason jasmine can't spy on such property, so the tests fail. Looking into this.